app/partials/equation_editor/equation_input.html

```diff
@@ -11,10 +11,11 @@
   text-align: center;
   cursor: pointer;
   background: white;">
-  &nbsp;
-</a>
+        &nbsp;
+    </a>
 </div>
-<div ng-show="editorMode == 'maths' && questionDoc.type && $root.userPreferences.BETA_FEATURE.EQN_TEXT_ENTRY">
+<div ng-show="editorMode == 'maths' && (isEquality || questionDoc.type && $root.userPreferences.BETA_FEATURE.EQN_TEXT_ENTRY)">
```

Comment:
I think the `questionDoc.type` in the `ng-show` was specifically to prevent the text entry from appearing on the `/equality` page, so this may need investigating and refactoring.